docs/creating/data_sources.md

```diff
@@ -23,6 +23,90 @@ In addition, when loading from file, there is the _required_ dimension `paramete
 This is a placeholder to point Calliope to the parameter name(s) that your tabular data is referring to.
 The values assigned as parameters will become the array names in your loaded model dataset ([`model.inputs`][calliope.Model.inputs]).
 
+## Structuring your CSV files
+
+### Include a header row
+
+CSV files in which tabular data is stored _must_ have at least one header row.
+If you do not have this header row, then the first row of data will still be taken to be dimension names, which Calliope will complain about.
```

Comment:
Maybe specify _how_ calliope will complain about this. Error, Warning, unexpected behavior?